teuthology/orchestra/run.py

```diff
@@ -268,15 +272,39 @@ def copy_to_log(f, logger, loglevel=logging.INFO, capture=None):
     # Work-around for http://tracker.ceph.com/issues/8313
     if isinstance(f, ChannelFile):
         f._flags += ChannelFile.FLAG_BINARY
-
     for line in f:
         if capture:
-            capture.write(line)
+            #out = stringify(line)
+            if PY3:
+                if isinstance(line, str):
+                    out = line.encode()
+                else:
+                    out = line.decode()
```

Comment:
why not use `six.ensure_text()` to coerce line to unicode string? so we don't need to conditionalize on PY3 and PY2.

Reply:
The code was written at before *ensure_text()* appeared in six. I need to refactor and make another round of testing.